gslib/tests/testcase/integration_testcase.py

```diff
@@ -1001,79 +1001,13 @@ def RunGsUtil(self,
       If only one return_* value was specified, that value is returned directly
       rather than being returned within a 1-tuple.
     """
-    # TODO(b/203250512) Remove this once all the commands are supported
-    # via gcloud storage.
-    if force_gsutil:
-      use_gcloud_storage = False
-    else:
-      use_gcloud_storage = config.getbool('GSUtil', 'use_gcloud_storage', False)
-    gcloud_storage_setting = [
-        '-o',
-        'GSUtil:use_gcloud_storage={}'.format(use_gcloud_storage),
-        '-o',
-        'GSUtil:hidden_shim_mode=no_fallback',
-    ]
-    cmd = [
-        gslib.GSUTIL_PATH, '--testexceptiontraces', '-o',
-        'GSUtil:default_project_id=' + PopulateProjectId()
-    ] + gcloud_storage_setting + cmd
-    if stdin is not None:
-      if six.PY3:
-        if not isinstance(stdin, bytes):
-          stdin = stdin.encode(UTF8)
-      else:
-        stdin = stdin.encode(UTF8)
-    # checking to see if test was invoked from a par file (bundled archive)
-    # if not, add python executable path to ensure correct version of python
-    # is used for testing
-    cmd = [str(sys.executable)] + cmd if not InvokedFromParFile() else cmd
-    env = os.environ.copy()
-    if env_vars:
-      env.update(env_vars)
-    # Ensuring correct text types
-    envstr = dict()
-    for k, v in six.iteritems(env):
-      envstr[six.ensure_str(k)] = six.ensure_str(v)
-    cmd = [six.ensure_str(part) for part in cmd]
-
-    # executing command - the setsid allows us to kill the process group below
-    # if the execution times out.  With python 2.7, there's no other way to
-    # stop the execution (p.kill() doesn't work).  Since setsid is not available
-    # on Windows, we just deal with the occasional timeouts on Windows.
-    preexec_fn = os.setsid if hasattr(os, 'setsid') else None
-    p = subprocess.Popen(cmd,
-                         stdout=subprocess.PIPE,
-                         stderr=subprocess.PIPE,
-                         stdin=subprocess.PIPE,
-                         env=envstr,
-                         preexec_fn=preexec_fn)
-    comm_kwargs = {'input': stdin}
-
-    def Kill():
-      os.killpg(os.getpgid(p.pid), signal.SIGKILL)
-
-    if six.PY3:
-      # TODO(b/135936279): Make this number configurable in .boto
-      comm_kwargs['timeout'] = 180
-    else:
-      timer = threading.Timer(180, Kill)
-      timer.start()
-
-    c_out = p.communicate(**comm_kwargs)
+    processed_command = util.GetGsutilCommand(cmd, force_gsutil=force_gsutil)
```

Comment:
What does "processed" mean in this case? If it just means some string generation has happened, might be a bit of a confusing word to put next to actual processes

Reply:
Changed to `full_gsutil_command`!